evo_robot:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100
 
  # lift_fr_controller:
  #   type: effort_controllers/JointVelocityController 
  #   joint: front_pndlm_link_2_fr_lift_link
  #   pid: {p: 0.0, i: 1.0, d: 0.0, i_clamp_min: -10, i_clamp_max: 10, antiwindup: True}
  # lift_fl_controller:
  #   type: effort_controllers/JointVelocityController 
  #   joint: front_pndlm_link_2_fl_lift_link
  #   pid: {p: 0.0, i: 1.0, d: 0.0, i_clamp_min: -10, i_clamp_max: 10, antiwindup: True}     
  # lift_br_controller:
  #   type: effort_controllers/JointVelocityController 
  #   joint: back_pndlm_link_2_br_lift_link
  #   pid: {p: 0.0, i: 1.0, d: 0.0, i_clamp_min: -10, i_clamp_max: 10, antiwindup: True}
  # lift_bl_controller:
  #   type: effort_controllers/JointVelocityController 
  #   joint: back_pndlm_link_2_bl_lift_link
  #   pid: {p: 0.0, i: 1.0, d: 0.0, i_clamp_min: -10, i_clamp_max: 10, antiwindup: True}
  
 
  wheel_fr_controller:
    type: effort_controllers/JointVelocityController 
    joint: fr_lift_link_2_fr_rim_link
    pid: {p: 0.0, i: 1.0, d: 0.0, i_clamp_min: -30.0, i_clamp_max: 30.0, antiwindup: True}
  wheel_fl_controller:
    type: effort_controllers/JointVelocityController 
    joint: fl_lift_link_2_fl_rim_link
    pid: {p: 0.0, i: 1.0, d: 0.0, i_clamp_min: -30.0, i_clamp_max: 30.0, antiwindup: True}
  wheel_br_controller:
    type: effort_controllers/JointVelocityController 
    joint: br_lift_link_2_br_rim_link
    pid: {p: 0.0, i: 1.0, d: 0.0, i_clamp_min: -30.0, i_clamp_max: 30.0, antiwindup: True}
  wheel_bl_controller:
    type: effort_controllers/JointVelocityController 
    joint: bl_lift_link_2_bl_rim_link
    pid: {p: 0.0, i: 1.0, d: 0.0, i_clamp_min: -30.0, i_clamp_max: 30.0, antiwindup: True}


  # roller_fr_controller:
  #   type: effort_controllers/JointVelocityController 
  #   joint: fr_rim_link_2_fr_roller_link
  #   pid: {p: 0, i: 60.0, d: 0.0, i_clamp_min: -100, i_clamp_max: 100, antiwindup: True}
  # roller_fl_controller:
  #   type: effort_controllers/JointVelocityController 
  #   joint: fl_rim_link_2_fl_roller_link
  #   pid: {p: 0, i: 60.0, d: 0.0, i_clamp_min: -100, i_clamp_max: 100, antiwindup: True}
  # roller_br_controller:
  #   type: effort_controllers/JointVelocityController 
  #   joint: br_rim_link_2_br_roller_link
  #   pid: {p: 0, i: 60.0, d: 0.0, i_clamp_min: -100, i_clamp_max: 100, antiwindup: True}
  # roller_bl_controller:
  #   type: effort_controllers/JointVelocityController 
  #   joint: bl_rim_link_2_bl_roller_link
  #   pid: {p: 0, i: 60.0, d: 0.0, i_clamp_min: -100, i_clamp_max: 100, antiwindup: True}
    
    

  # gazebo_ros_control:
  #   pid_gains:
  #     base_link_rim_wheel_fr_joint: {p: 5.0, i: 0.0, d: 0.0}
  #     base_link_rim_wheel_fl_joint: {p: 5.0, i: 0.0, d: 0.0}
  #     base_link_rim_wheel_br_joint: {p: 5.0, i: 0.0, d: 0.0}
  #     base_link_rim_wheel_bl_joint: {p: 5.0, i: 0.0, d: 0.0}